routes/accounts.js

```diff
@@ -80,10 +81,22 @@ function accountRoutes (server, options, next) {
     handler: function (request, reply) {
       var sessionId = toBearerToken(request)
 
-      return accounts.findAll({
-        db: options.db,
-        bearerToken: sessionId,
-        include: request.query.include
+      admins.validateSession(sessionId)
+
+      .catch(function (error) {
+        if (error.name === dbErrors.MISSING_DOC.name) {
```

Comment:
I think status codes are enough. The error messages won’t change, they are the same as http://couchdb.apache.org/ throws them, and the error status would change even less. Maybe we can compare against the status code in the other places, too, to make it coherent. But let’s do that in a follow up PR?